layers/meta-resin-raspberrypi/recipes-bsp/u-blox-modeswitch/files/u-blox-switch.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+while true; do
+        echo -e "AT+UUSBCONF=2,\"ECM\"\r\n" > /dev/ttyACM0
+        sleep 2
+        echo -e "AT+CFUN=1,1\r\n" > /dev/ttyACM0
+        sleep 20
+        lsusb | grep U-Blo | grep 1143
```

Comment:
You could do the if on the command directly. In this way, it will not fail if we run it with `-e`.

Reply:
I don't see a problem with the way it's implemented now, and I don't understand who is to be run with -e